src/main/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/PersistenceService.java

```diff
@@ -150,12 +151,18 @@ public ActorCacheEntity getActorCacheEntity(String actorId) {
         return actorCacheRepository.findByActorId(actorId);
     }
 
-    public List<RoleAssignment> getAssignmentsByProcess(String process, String reference, String status) {
-        Set<HistoryEntity> historyEntities = historyRepository.findByReference(process, reference, status);
-        //convert into model class
-        return historyEntities.stream().map(historyEntity -> persistenceUtil
-            .convertHistoryEntityToRoleAssignment(historyEntity)).collect(
-            Collectors.toList());
+    public List<RoleAssignment> getAssignmentsByProcess(String process, String reference) {
+        Set<RoleAssignmentEntity> roleAssignmentEntitySet =
+            roleAssignmentRepository.findByProcessAndReference(process.toUpperCase(), reference.toUpperCase());
+        List<RoleAssignment> roleAssignmentList = roleAssignmentEntitySet.stream().map(entity -> persistenceUtil.convertEntityToRoleAssignment(entity))
+            .collect(Collectors.toList());
+
+        for(RoleAssignment assignment: roleAssignmentList) {
+            assignment.setStatus(Status.LIVE);
+        }
```

Comment:
we don't need to set the status on the live records. we can update the LLD if required.